### ISSUE TITLE
meintin Issues tracker on "Support" tab of Kranc website

### ISSUE THREAD

**Issue by rhaas80** [ianhinder/Kranc#149]
It maybe be good to link to this issue tracker on the Support page: http://kranccode.org/support.html as well as on the Development page http://kranccode.org/development.html

**Comment by ianhinder** [comment]
Yes please! You have write access; the site is on [GitHub Pages](https://github.com/ianhinder/Kranc/tree/gh-pages).
